sqlstore/src/main/resources/db/migration/V8000__sample_class_alias.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE SampleClass ADD COLUMN sampleSubcategory varchar(50) NOT NULL DEFAULT '';
```

Comment:
Can we use null instead of empty string?

Reply:
We can. I didn't because most of the logic is set up as `getSampleCategory().equals(whatever)` and I didn't want to make all of those null-safe.

Reply:
We use null everywhere else though, and since we often have objects not fully populated (often only the ID set), making it all null safe will likely prevent NPEs at some point

Reply:
Okay. Fixed.